### PR TITLE
Add static templates with linters

### PR DESCRIPTION
**Pull request by josuehoenicka**
- Complete HTML & CSS Task
- All linters tests passed
- Use the default README.md structure
- Hidden node_modules folders and test.md file

### CODE REVIEW

File: .github/workflows/linters.yml
Comment (ouasamine) [inline .github/workflows/linters.yml:1]: - [x] Nice work creating this config file, it is essential for the GitHub actions to work. However, you forgot to add its content and that's why the GitHub actions (linters checks) are currently not working, please copy the content from [here](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) to your file.

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @josuehoenicka 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Nice commit messages, they are descriptive and concise ✔️ 👏🏽 
- Good PR title and description 👍🏽 

## Required Changes ♻️
- [x] Nice work adding the README file using the correct template, however, the link in the license section is currently not functional, because the link is pointing to a file named `LICENSE` which doesn't exist in your repo, please change the name of your `MIT.md` file to `LICENSE` or change the link in your README to fix this issue.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @josuehoenicka ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [x] Well done on trying to make some changes 💯 but the link of the License is looking for a file named `LICENSE` while your file is named `LICENSE.md`, look to fix that so that it can work 🤓 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- [x] The Microverse logo is nice 👌 but you could consider removing it or even adding your own instead maybe.
- [x] Some sections seem to be empty in the README, such as `Setup`, `Run Tests` and `Deployment`. If you're not sure what to put in them you can put `N/A` or specify that this section is not relevant to this project yet. Also, the `Usage` is duplicated 2️⃣ 👀 

- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by LionRouge1 — APPROVED** [review]
Hi @josuehoenicka , 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes,
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**